x/wasm/types/events.go

```diff
@@ -6,28 +6,26 @@ const (
 	// CustomContractEventPrefix contracts can create custom events. To not mix them with other system events they got the `wasm-` prefix.
 	CustomContractEventPrefix = "wasm-"
 
-	EventTypeStoreCode            = "store_code"
-	EventTypeInstantiate          = "instantiate"
-	EventTypeExecute              = "execute"
-	EventTypeMigrate              = "migrate"
-	EventTypeUpdateAdmin          = "update_admin"
-	EventTypeClearAdmin           = "clear_admin"
-	EventTypeUpdateContractStatus = "update_contract_status"
-	EventTypePinCode              = "pin_code"
-	EventTypeUnpinCode            = "unpin_code"
-	EventTypeSudo                 = "sudo"
-	EventTypeReply                = "reply"
-	EventTypeGovContractResult    = "gov_contract_result"
+	EventTypeStoreCode         = "store_code"
+	EventTypeInstantiate       = "instantiate"
+	EventTypeExecute           = "execute"
+	EventTypeMigrate           = "migrate"
+	EventTypeUpdateAdmin       = "update_admin"
+	EventTypeClearAdmin        = "clear_admin"
+	EventTypePinCode           = "pin_code"
+	EventTypeUnpinCode         = "unpin_code"
+	EventTypeSudo              = "sudo"
+	EventTypeReply             = "reply"
+	EventTypeGovContractResult = "gov_contract_result"
 )
 
 // event attributes returned from contract execution
 const (
 	AttributeReservedPrefix = "_"
 
-	AttributeKeyContractAddr   = "_contract_address"
-	AttributeKeyCodeID         = "code_id"
-	AttributeKeyCodeIDs        = "code_ids"
-	AttributeKeyResultDataHex  = "result"
-	AttributeKeyFeature        = "feature"
-	AttributeKeyContractStatus = "contract_status"
+	AttributeKeyContractAddr  = "_contract_address"
```

Comment:
no need under bar?

Reply:
This is not my change. 

https://github.com/CosmWasm/wasmd/blob/e714fdf3b44a93462232648ab670ff611c381ce1/x/wasm/types/events.go#L24